drivers/sensor/fxos8700/fxos8700.h

```diff
@@ -5,7 +5,11 @@
  */
 
 #include <zephyr/drivers/sensor.h>
+#if DT_ANY_INST_ON_BUS_STATUS_OKAY(spi)
+#include <zephyr/drivers/spi.h>
+#elif DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c)
 #include <zephyr/drivers/i2c.h>
+#endif
```

Comment:
```suggestion
#if DT_ANY_INST_ON_BUS_STATUS_OKAY(spi)
#include <zephyr/drivers/spi.h>
#endif

#if DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c)
#include <zephyr/drivers/i2c.h>
#endif
```

Reply:
Acknowledged

Reply:
@MaureenHelm .. 
Though I understand your comments for removing the #elif but fyi..
We got inspired from following sensor drivers, where similar #if, #elif constructs are used in a similar scenario-
- drivers/sensor/iis2dh/iis2dh.c
- drivers/sensor/lis2dw12/lis2dw12.c
 Do you see any difference between the above 2 drivers and our case or do these also need to change?